.github/workflows/format.yaml

```
@@ -0,0 +1,13 @@
+name: Check formatting with Prettier
+on: [push, pull_request]
+
+jobs:
+  check-format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+      - name: Check formatting
+        run: |
+          shopt -s globstar
+          npx prettier --check **/*.js
```

How about also formatting TypeScript files (e.g. see [here](https://github.com/espruino/BangleApps/tree/master/apps/banglerun/src)) and json files?

---

Or simply let prettier take care of all supported formats?

```
npx prettier --check .
```

---

Alright, I'll do that.
```suggestion
          npx prettier --check .
```

---

Running Prettier has discovered a number of invalid HTML problems. I'll attempt to fix them so it can run.